### PR TITLE
Add datapackage pipelines post

### LINKED ISSUE #487: Post: datapackage-pipelines
https://github.com/frictionlessdata/datapackage-pipelines

### PR DESCRIPTION
**Pull request by danfowler**
Closes #487

### CODE REVIEW

File: blog/_posts/2017-02-27-datapackage-pipelines.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+author: Adam Kariv
+title: Data Package Pipelines
+username: akariv
+---
+
+*[datapackage-pipelines][dpp] is the newest part of the Frictionless
+ Data toolchain.  It is a framework for defining data processing steps
+ declaratively to generate self-describing Data Packages.*
+
+[OpenSpending][os] is an open database for
```
Comment (tfmorris) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: This doesn't flow very well from "DPP is" to "OpenSpending is". Perhaps a smoother transition would be something like "We were searching for an ETL solution for OpenSpending" or something similar which doesn't put the focus first on OpenSpending.

File: blog/_posts/2017-02-27-datapackage-pipelines.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+2. **Input and output works through streaming data.**
+
+    While this means processing nodes have limited flexibility, it
+    also means they must adhere to strict use of computing
+    resources. This constraint allows us to deploy processing flows
+    more easily, without having to worry about a processing node
+    taking too much memory or disk space.
+
+3. **Like OpenSpending, we are based on the Data Package.**
```
Comment (tfmorris) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: Is this the primary reason? If so, put it first.

File: blog/_posts/2017-02-27-datapackage-pipelines.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+use cases:
+
+![Pipelines for something other than data](/img/posts/dpp-pipelines.jpg)
+
+*[Pipelines](https://www.flickr.com/photos/loopkid/359144645) for
+ something other than data -
+ [Stefan Schmidt](https://www.flickr.com/photos/loopkid/) -
+ [CC BY-NC 2.0](https://creativecommons.org/licenses/by-nc/2.0/)*
+
+1. **Processing flows (or 'pipelines') are defined in a configuration
```
Comment (tfmorris) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: Config files are just code by a different name

**Comment by tfmorris** [comment]
Often these build vs "buy" (aka steal) tradeoffs have some positives and some negatives which are weighed against each other. Is it the case here that there are no negatives and only positives? I don't see any negatives mentioned.

**Comment by danfowler** [comment]
Thanks @todrobbins!

Thanks @tfmorris really insightful commentary here.  I will do a revision based on your comments before publishing next week.  One negative @akariv highlighted was the limited flexibility of this approach.  Did that not come across (my fault due to editing)?

File: blog/_posts/2017-02-27-datapackage-pipelines.md (hunk 3 above)
Comment (danfowler) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: Noted :).

File: blog/_posts/2017-02-27-datapackage-pipelines.md (hunk 2 above)
Comment (danfowler) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: Fixed.

File: blog/_posts/2017-02-27-datapackage-pipelines.md (hunk 1 above)
Comment (danfowler) [inline blog/_posts/2017-02-27-datapackage-pipelines.md]: I think I fixed this.  Thanks!

**Pull request merged**